frontend/src/lib/router.tsx

```diff
@@ -699,10 +714,14 @@ const defaultRoutes: {
     ),
   },
   settingsCluster: {
-    path: '/settings',
+    path: '/settings/cluster',
```

Comment:
Instead of replacing this route. Keep the route (but adapt the sidebar), then add a new route that is essentially the one you replaced it with (/settings/cluster). Because we don't know if some plugins out there refer to this route, in which case we'd be breaking their use-cases.

Reply:
Done